helix-core/src/main/java/org/apache/helix/HelixAdmin.java

```diff
@@ -228,6 +228,13 @@ void addResource(String clusterName, String resourceName, int numPartitions, Str
    */
   void dropInstance(String clusterName, InstanceConfig instanceConfig);
 
+  /**
+   * Purge offline instances from a cluster
+   * @param clusterName
+   * @param customizedPurgeMap
+   */
+  void purgeOfflineInstances(String clusterName, Map<String, String> customizedPurgeMap);
```

Comment:
Should not be the API looks like:  purgeOfflineInstances(clusterName, long timeout), which will delete all instances that have been offline longer than timeout?